docker-compose.yml

```diff
@@ -0,0 +1,13 @@
+web:
+  build: .
+  ports:
+    - "8888:8888"
```

Comment:
I'm not sure either, but my thinking is that this is the -p flag and we would need this for docker-machine to expose the server port to the hose